### DISCUSSION TITLE
Effects editor

### DISCUSSION THREAD

**Discussion by WeylonSantana** [AscensionGameDev/Intersect-Engine#1870] [Ideas]
![image](https://github.com/AscensionGameDev/Intersect-Engine/assets/63019821/7215741c-1a8b-496a-a641-8972853b4173)

Trying to base myself on that, but focusing 100% on the intersection and the possibilities of what can be done

**Comment by LunamOdium** [comment]
Yes Bro!!!!! I cant wait!!!!!

**Comment by PyroTech03** [comment]
This would be amazing.

**Comment by iceicebabyy** [comment]
Could this be added? Please, it is so important to have greater customization of spells. I would also add the possibility of reviving others or oneself.

**Reply by WeylonSantana** [reply]
> Could this be added? Please, it is so important to have greater customization of spells. I would also add the possibility of reviving others or oneself.

It's easier said than done, we have to think about all the possibilities and scopes, and things for the original engine, we must also think about user migration, so that users don't lose their magic, something like this will take a long time to be redone.
If it was something just for one user, we would tell him to delete all the spells and do it again, we can't tell everyone who is using the intersect to do this
